doc/syntax.org

```diff
@@ -0,0 +1,43 @@
+#+TITLE:   Markup Syntax Guide for Pancacke(d)
+#+AUTHOR:  Scott Wang
+#+STARTUP: align indent
+#+OPTIONS: H:5
+
+
+* Overview
```

Comment:
Currently I assume the top level headline *MUST* be in uppercase, should we keep this?